lib/ecto/query.ex

```diff
@@ -95,6 +95,13 @@ defmodule Ecto.Query do
   For this reason, we will use schemas on the remaining examples but
   remember Ecto does not require them in order to write queries.
 
+  ## `nil` comparison
+
+  Nil comparison with a infix operators is forbidden as it is unsafe
```

Comment:
s/Nil/`nil`

Reply:
Lets remove the mention of infix operators, it doesn't add much and can potentially confuse.